tests/app/core/models/session-model.spec.ts

```diff
@@ -29,6 +29,7 @@ describe('Session Model', () => {
       of(undefined)
         .pipe(assertSessionDataIntegrity())
         .subscribe({
+          next: fail,
```

Comment:
Actually to standardize the coding style should we use the `() => fail()`? And maybe we can combine the `error` one to a single line also instead of multi-line since there is still space?